test/unit/spec/v2/models/AppState_spec.js

```diff
@@ -118,7 +118,7 @@ describe('v2/models/AppState', function() {
   });
 
   describe('shouldReRenderView', () => {
-    it('rerender view should be false if stateHandle are different', () => {
+    it('rerender view should be true if stateHandle are different', () => {
```

Comment:
if only stateHandle is different, we don't expect to re-render (but only refresh `appState`), do we?

Reply:
i will remove this